sipa/forms.py

```diff
@@ -275,6 +279,87 @@ class ChangeMACForm(FlaskForm):
     )
 
 
+def validate_iban(form, field: StringField) -> None:
+    try:
+        IBAN(field.data, validate_bban=True)
+    except SchwiftyException as e:
+        # TODO Figure out decent error messages
```

Comment:
placeholder comment so you don't forget this `TODO`

Reply:
I think displaying something like "Invalid IBAN" makes the most sense here, since the information provided by _schwifty_ is most likely too technically for most (if not all) users.